pkg/routes/routes.go

```diff
@@ -21,14 +21,15 @@ type DMSRouter struct {
 // Pingdom is the endpoint that pingdom will query, it returns 200 if all the checks are within the heartbearExpiration
 // and it returns 500 if any of the checks are outside of the heartbearExpiration
 func (r *DMSRouter) Pingdom(c echo.Context) error {
-	all, err := r.Store.All()
+	all, err := r.Store.AllCheckins()
 	if err != nil {
 		logrus.WithError(err).Error("unable to retrieve all stored data from etcd")
 	}
 	for k, v := range all {
-		if v.Last != (time.Time{}) {
-			if time.Since(v.Last) > (r.HeartbeatExpiration) {
+		if v != (time.Time{}) {
```

Comment:
nit: clearer if `v.IsZero()` method is used

Reply:
`IsZero()` is actually different than `time.Time{}`

Reply:
Time.IsZero checks whether the time is zero ignoring the timezone information. Checking against `== time.Time{}` also checks that the timezone is unset.